src/test/java/com/targomo/client/api/request/TimeVectorRequestTest.java

```diff
@@ -0,0 +1,173 @@
+
+public class TimeVectorRequestTest extends RequestTest {
+
+	@Test
+	public void get_success() throws Exception {
+		// Mock success response
+		when(sampleResponse.getStatus()).thenReturn(Response.Status.OK.getStatusCode());
+
+		// Get sample json when success response is queried
+		InputStream resourceAsStream = getClass().getClassLoader().getResourceAsStream("data/TimeVectorResponseSample.json");
+		String sampleJson = IOUtils.toString(resourceAsStream, Charset.forName("UTF-8"));
+		when(sampleResponse.readEntity(String.class)).thenReturn(sampleJson);
+
+		// Make the call
+		TimeVectorResponse response = TimeVectorRequest.executeRequest(mockClient, getTravelOptions());
+
+		// Check result
+		assertEquals("ok", response.getCode());
+		//TODO
+//		assertEquals(13, response.getRequestTimeMillis());
+//
+//        assertNotNull(response.getData());
+//		assertNotNull(response.getData().getNodes());
+//        assertFalse(response.getData().getNodes().isEmpty());
+//        assertNotNull(response.getData().getLayers());
+//        assertFalse(response.getData().getLayers().isEmpty());
+//        assertNotNull(response.getData().getEdges());
+//        assertFalse(response.getData().getEdges().isEmpty());
+//        assertNotNull(response.getData().getSupportingPoints());
+//        assertFalse(response.getData().getSupportingPoints().isEmpty());
+	}
+//
+//    @Test
+//    @Ignore("To test this you will have to fill in your API key first")
+//    public void get_success_API_test() throws Exception {
+//
+//	    String apiKey = "YOUR_API_KEY_HERE";
+//
+//	    //prepare travelOptions
+//        TravelOptions tO = getTravelOptions();
+//        tO.setServiceKey(apiKey);
+//        tO.setServiceUrl("https://service.route360.net/westcentraleurope");
+//
+//        // Make the call
+//        Client client = ClientBuilder.newClient();
+//        client.register(new GZipEncoder());
+//        MultiGraphRequest<MultiGraphJsonResponse> request = new MultiGraphRequest<>(client,tO,MultiGraphJsonResponse.class);
+//        MultiGraphJsonResponse response = MultiGraphRequest.executeRequestJson(client,tO);
+//
+//        System.out.println(request.toCurl());
+//
+//        assertNotNull(response.getData());
+//    }
+//
+//	@Test
+//	public void get_gateway_timeout() throws Exception {
+//		when(sampleResponse.getStatus()).thenReturn(Response.Status.GATEWAY_TIMEOUT.getStatusCode());
+//
+//        // Make the call
+//        MultiGraphResponse response = MultiGraphRequest.executeRequestJson(mockClient, getTravelOptions());
+//
+//        assertEquals("gateway-time-out", response.getCode());
+//	}
+//
+//	@Test(expected = TargomoClientException.class)
+//	public void get_exception() throws Exception {
+//		when(sampleResponse.getStatus()).thenReturn(Response.Status.INTERNAL_SERVER_ERROR.getStatusCode());
+//
+//        // Make the call
+//        MultiGraphRequest.executeRequestJson(mockClient, getTravelOptions());
+//	}
+//
```

Comment:
Shouldn't we remove the source code commented out from TimeVectorRequestTest.java?

Reply:
fixed